browser/net/brave_common_static_redirect_network_delegate_helper_unittest.cc

```diff
@@ -11,7 +11,6 @@
 #include "base/command_line.h"
 #include "brave/browser/net/url_context.h"
 #include "brave/common/network_constants.h"
-#include "brave/components/brave_component_updater/browser/switches.h"
```

Comment:
`component_updater_url_constants` not used?

Reply:
resolved!